cmd/notary-server/config.go

```diff
@@ -21,9 +23,21 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/spf13/viper"
 	"golang.org/x/net/context"
-	"gopkg.in/dancannon/gorethink.v2"
+	gorethink "gopkg.in/dancannon/gorethink.v2"
 )
 
+// gets the required gun prefixes accepted by this server
+func getRequiredGunPrefixes(configuration *viper.Viper) ([]string, error) {
+	prefixes := configuration.GetStringSlice("repositories.gun_prefixes")
+	for _, prefix := range prefixes {
+		p := path.Clean(strings.TrimSpace(prefix))
+		if p+"/" != prefix || strings.HasPrefix(p, "/") || strings.HasPrefix(p, "..") {
```

Comment:
I'm wondering if we also want to enforce the leading `/` and `..` cases for GUNs if the config does not specify any prefixes?

Unless I'm missing something, notary could still create trust data for GUNs like `/gun` and `../../gun` if we never get into this loop